rest-tests/src/test/resources/features/operator-deployment.feature

```diff
@@ -273,16 +273,34 @@ Feature: Operator Deployment
     Then check that the deployment config "syndesis-server" contains variables:
       | OPENSHIFT_MANAGEMENT_URL_FOR3SCALE | asdf |
 
+  @operator-backup-restore
+  @operator-backup-multiple
+  Scenario: Syndesis Operator - Multiple backups
+    Given create sample bucket on S3 with name "syndesis-backup"
+      And clean backup S3 bucket
+    When create pull secret for backup
+      And deploy Syndesis CR from file "spec/backup-multiple.yml"
+      And wait for Syndesis to become ready
+      And wait for backup
+    When clean backup S3 bucket
+    Then sleep for jenkins delay or "200" seconds
+      And verify that there are 3 backups in S3
+
   @ENTESB-12114
   @ENTESB-12846
+  @ENTESB-13046
   @operator-backup-restore
-  Scenario: Syndesis Operator - Backup and Restore
-    Given deploy ActiveMQ broker
+  @operator-backup-restore-procedure
+  Scenario Outline: Backup and Restore
```

Comment:
please also add `<method>` and `<type>` to the scenario name so that it shows up in the report

Reply:
done